aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -674,6 +938,49 @@ module aptos_framework::coin {
         let BurnCapability<CoinType> {} = burn_cap;
     }
 
+    fun mint_internal<CoinType>(amount: u64): Coin<CoinType> acquires CoinInfo {
```

Comment:
inline specs make this hard to read, can we move them out?

Reply:
could we leave this to the prover/spec team...?